### ISSUE TITLE
bug(mat-date-range-picker): In windows high contrast mode, focus indicator is not present for selected dates

### ISSUE THREAD

**Issue by sourabhsoni0711** [angular/components#22873]
#### Reproduction

Use StackBlitz to reproduce your issue: 
- Components: https://stackblitz.com/fork/components-issue
- Harnesses: https://stackblitz.com/fork/harness-issue

Steps to reproduce:
1 .Enable High contrast mode in windows machine
2. Open date range picker in example 1 https://material.angular.io/components/datepicker/examples
3. Observe that 'selected date' and 'Down pointing arrow' are not visible in HC mode and does not have focus indicator. 
 
 
#### Expected Behavior

In HC mode, all the elements should be visible and also visible focus indicator should be present for all controls

#### Actual Behavior

'Down pointing arrow' is invisible and the 'selected dates'  does not have a visible focus indicator.

#### Environment

- Angular: 11.0.3
- CDK/Material: 11.0.3
- Browser(s): Firefox 88 
- Operating System (e.g. Windows, macOS, Ubuntu): Windows 10 Enterprise

**Comment by crisbeto** [comment]
I'm submitting a fix for the focus indication on selected dates. The issue with down arrows is known and it's something that we'd have to tackle across the entire project.
